.github/workflows/_swift.yml

```diff
@@ -71,7 +71,7 @@ jobs:
           API_KEY_ID: "${{ secrets.APPLE_APP_STORE_CONNECT_API_KEY_ID }}"
           API_KEY: "${{ secrets.APPLE_APP_STORE_CONNECT_API_KEY }}"
       - run: ${{ matrix.upload-script }}
-        if: ${{ github.event_name == 'workflow_dispatch' || (github.ref == 'refs/heads/main' && contains(github.event.head_commit.modified, 'elixir/VERSION')) }}
```

Comment:
I think we need the same fix in other workflows too! Never knew what this elixir version stuff was about.

Reply:
Yeah I'll update the Android workflow to have it